lib/routes.js

```diff
@@ -0,0 +1,21 @@
+var locations = require('./api/v1/locations');
+
+module.exports = [
+  {
+    method: 'GET',
+    path: '/api/v1/locations',
+    handler: locations.methods.all
+  },
+  {
+    method: 'POST',
+    path: '/api/v1/locations',
+    handler: locations.methods.create
+  },
```

Comment:
I'm a fan of the way Hapi works. So far it seems pretty streamlined. Of course we haven't done anything yet....